src/main.ts

```diff
@@ -201,29 +201,24 @@ async function main(): Promise<void> {
 
         // Iterate through all open issues and pull requests
         for await (const page of octokit.paginate.iterator(
-          octokit.issues.listForRepo,
+          octokit.rest.issues.listForRepo,
           { owner, repo, since: offsetDate.toISOString() },
         )) {
           for (const issue of page.data) {
-            const {
-              body,
-              number,
-              html_url,
-            } = issue as octokitTypes.IssuesGetResponseData;
```

Comment:
I was not able to get this destructuring to work while respecting Typescript. `octokitTypes.IssuesGetResponseData` is not a type exported by octokitTypes any more. I believe the equivalent type can be retrieved with this formulation:
```
octokitTypes.Endpoints["GET /repos/{owner}/{repo}/issues"]["response"];
```

However, even with that I had difficulty satisfying Typescript.